indy_node/server/request_handlers/read_req_handlers/get_nym_handler.py

```diff
@@ -0,0 +1,5 @@
+from indy_node.server.request_handlers.read_request_handler import ReadRequestHandler
+
+
+class GetNymHandler(ReadRequestHandler):
```

Comment:
I think we need to implement all Domain ReadRequestHandlers in the scope of INDY-1857

Reply:
ok